src/main.rs

```diff
@@ -1,6 +1,6 @@
 // We don't need to hunt of unused imports on windows, as they are harmless
 #![cfg_attr(windows, allow(unused_imports))]
-#![type_length_limit="2097152"]
+#![type_length_limit="2134214"]
```

Comment:
Just curious: is this for linting?

Reply:
Not exactly. Looks like new compiler exposed a limit (or just created a longer type) and we needed to bump it. Now we bump it again as we added some longer type (i.e. because of deeper nesting of futures is the most likely case)